website/content/docs/developers/release-process-maven.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Publish Release to Maven Central
```

Comment:
Can we make this a broader-scoped doc `release-process.md` that covers this but also:
- how to update version
- how to tag
- test process
- build process
- maven process

Some of those sections can be placeholders.